### ISSUE TITLE
New rule for handling operators and line breaks [$15]

### ISSUE THREAD

**Issue by goloroden** [eslint/eslint#1405]
I would like to suggest a rule that defines how to handle operators and line breaks, i.e. whether it's

    var sum = (a +
      b);

or

    var sum = (a
      + b);

Basically the rule should have two options: `ops-before-linebreaks` and `ops-after-linebreaks` (or something similar, the naming isn't good ;-)).

**Comment by ValeriiVasin** [comment]
Is it possible to handle last dot notation with this rule? Or maybe another rule exist to forbid syntax like:

```javascript
someObject.
  someMethod()
```

when the new line is after the dot?

**Comment by nzakas** [comment]
The dot isn't an operator, so we would exempt from this rule.

**Comment by feross** [comment]
@nzakas Hm, what is the dot considered if not an operator? I'm curious.

**Comment by lo1tuma** [comment]
@feross The dot is a token within of a `MemberEpression`.

**Comment by feross** [comment]
@ilovejary Are you still working on this? Any updates?

**Comment by nzakas** [comment]
No one has said they have started working on this, the issue is open for anyone to take.

**Comment by BenoitZugmeyer** [comment]
Working on this.
